doc/mgr/ssh.rst

```diff
@@ -0,0 +1,32 @@
+================
+SSH orchestrator
+================
+
+The SSH orchestrator is an orchestrator module that does not rely on a separate
+system such as Rook or Ansible, but rather manages nodes in a cluster by
+establishing an SSH connection and issuing explicit management commands.
+
+Orchestrator modules only provide services to other modules, which in turn
+provide user interfaces.  To try out the SSH module, you might like
+to use the :ref:`Orchestrator CLI <orchestrator-cli-module>` module.
+
+Requirements
+------------
+
+- The Python `remoto` library version 0.35 or newer
```

Comment:
This is similar to the rook orchestrator, which depends on `kubernetes-client`: we're implicitly adding new dependency here. This feels as if we should make the ssh orchestrator a separate package. @dmick ?

Reply:
yes.. let's get https://github.com/ceph/ceph/pull/25977 merged first, though.